speckle/resources/objects.py

```diff
@@ -55,13 +54,14 @@ def __init__(self, session, basepath, me):
            
         self.schema = SpeckleObject
 
-    def list(self):
+    def list(self, query=None):
         """List all Speckle objects
         
         Returns:
             list -- A list of Speckle object data class instances
         """
-        return self.make_request('list', '/')
+        query_string = self.make_query(query)        
+        return self.make_request('list', '/' + query_string)
```

Comment:
Yep I think we should pass the query arguments as a dict or something and then pass them into the `requests` call as a param object.

Reply:
Ok, that makes a lot of sense. Your query string builder was kind of doing that, so I just put it in a reusable function. Will take a look at the `requests` docs...

Reply:
Changed to use `requests` query params.